contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -80,11 +80,11 @@ In this case, only the `root` component can be accessed in Vue while the rest ar
 <b>Step 4:</b> Pass the message to the HTML file using double curly brackets.
 ```HTML
   <div id="root">
-    <h2>{\{message}\{</h2>
+    <h2>{\{message}\}</h2>
   </div>
 ```
 
-<b>Step 5:</b> Open the brower and we will see "Hello World" being displayed:
+<b>Step 5:</b> Open the browser and we will see "Hello World" being displayed:
 > <h2>Hello World</h2>
```

Comment:
Maybe replace this with an actual screenshot?

Reply:
don't think this is necessary tho, the point to bring across is that `Hello World` will be displayed